config/config.go

```diff
@@ -360,6 +363,22 @@ func (cfg Config) ApplicationName() string {
 	return cfg.General.ApplicationName
 }
 
+// VoiceName will return the Twilio.VoiceName value
+func (cfg Config) VoiceName() string {
+	if cfg.Twilio.VoiceName == "" {
+		return "Polly.Joanna-Neural"
+	}
+	return cfg.Twilio.VoiceName
+}
+
+// VoiceLanguage will return the Twilio.VoiceLanguage value
+func (cfg Config) VoiceLanguage() string {
+	if cfg.Twilio.VoiceLanguage == "" {
+		return "en-US"
+	}
+	return cfg.Twilio.VoiceLanguage
+}
```

Comment:
The default should be empty so that the voice config continues to be whatever is configured in the Twilio account unless a value is set

Reply:
adjusted as per your request